hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/shell/Display.java

```diff
@@ -217,8 +216,8 @@ protected void processPath(PathData item) throws IOException {
 
   protected class TextRecordInputStream extends InputStream {
     SequenceFile.Reader r;
-    Writable key;
-    Writable val;
+    Object key;
```

Comment:
What is the use case to not base it on Writable? The whole principal is to use Writable.

Reply:
Hi @goiri. Thanks for your review. 

Currently - The Display command (fs -text) currently reads only Writable-based SequenceFiles. This isn't necessary to do, and prevents reading non-Writable-based serialization in SequenceFiles from the shell.